### PR TITLE
Capstone the First

### PR DESCRIPTION
**Pull request by Banstein**
- Add HTML file, Add the design skeleton, Add dynamic links.
- Add CSS file, Add responsive design for desktop and mobile.
- Add JS file, Add popup menu for mobile, objects data type for dom-manipulation and HTML element injection.
- Add images, for backgrounds and partners, speakers.
- When you click or 'tab' links it will redirect you to the mentioned sections.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+      <img src="images/partners/facebook.png" alt="Missed Image">
+      <img src="images/partners/amazon.png" alt="Missed Image">
+      <img src="images/partners/airbnb.png" alt="Missed Image">
+      <img src="images/partners/mozilla.png" alt="Missed Image">
+    </div>
+  </footer>
+  <div class="copy-write hide-mobile">
+    <img src="images/riseup.png" alt="Missed Image" class="copy-img">
+    <p class="copy-text">2021 Egypt Riseup Summit Some Rights reserved.</p>
+  </div>
```
Comment (oxenprogrammer) [inline index.html]: **Suggested Changes**
- [ ] This should be part of the footer HTML semantics. Let us also reduce the size of the logo on the desktop version.
- [ ] The footer should have a dark background on the desktop version. Currently on your design it's white background in the home page.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  </header>
+  <article class="home">
+    <div class="home-container">
+      <p class="home-welcome">"Hello! Sharing world"</p>
+      <h1 class="home-intro">Egypt RISEUP<br>GLOBAL SUMMIT<br>2022</h1>
+      <p class="home-desc">A joyful celebration believing in the value of openness and sharing, creating change with pepole form over 80 countries is taking place in october, in Egypt.</p>
+      <p class="home-time">2022.1.15(THU) ~ 16(FRI)</p>
+      <p class="home-lctn">@American University of Cairo, Conference Center </p>
+    </div>
+  </article>
```
Comment (oxenprogrammer) [inline index.html:50]: **Suggested Changes**
- [ ] Let us please increase the font size of the `h1` tag.
- [ ] After the above, let us please increase the font size of the `p` tags relative to the `h1` tag.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,627 @@
...
+  }
+
+  .program-list li h4 {
+    width: 24%;
+  }
+
+  .program-list li p {
+    width: 48%;
+  }
+}
```
Comment (oxenprogrammer) [inline style.css]: **Suggested Changes**
- [ ] We should use a mobile-first approach. This is a desktop-first media query. Let us effect this change and follow a mobile first approach. You may research on the benefits for mobile first.

**Review by oxenprogrammer — CHANGES_REQUESTED** [review]
Hi @Banstein, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Suggested changes
- [ ] The video presentation should contain a `piece of code` you're proud of writing.
- [ ] Currently the mobile version is distorted, let us fix it. I think removing the `horizontal` scroll will help. Check the attached video of the current behavior. 

https://user-images.githubusercontent.com/7220167/147376987-21702220-80cf-431c-b368-59468e9aa80d.mp4


_Check the comments under the review for more suggested changes._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html (hunk 1 above)
Comment (Banstein) [reply index.html]: partner section is dark background as the design, also Copywrite is white background exact like the design too on homepage.

**Review by MrBrN197 — APPROVED** [review]
## Hi, @Banstein  👋

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**